CHANGELOG-0.7.md

```diff
@@ -4,7 +4,7 @@
 
 ### Added
 
-- Minor logging improvements added while fixing issue [#1424](https://github.com/epiphany-platform/epiphany/issues/1424).
+- [#1424](https://github.com/epiphany-platform/epiphany/issues/1424) - Minor logging improvements added while fixing issue
```

Comment:
Dont think this needed to be changed. This was just an addition I made while fixing #1424 and not #1424 itself which is mentioned in the fixed list.

Reply:
Sure.